GoogleForCreators/web-stories-wp
# Eyedropper: handle rgb string when getting the eyedropper color

## Context



## Summary



## Relevant Technical Choices
- Adds handling colour string in rgb.
- Removes part of the patch that breaks the logic for FF and Safari.


## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Add a text element (or any other that has styling with colors)
2. Use eyedropper for changing color: test in the colour picker, Style tab and also in the Floating edit menu of the element.
3. Verify eyedropper works as expected.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #12560
